app/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -82,52 +81,25 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
   });
 
   /**
-   * Handles the form submission.
-   * @param data - The form data.
+   * Handles the weekly picks
+   * @param data - data of the pick
    * @returns {void}
    */
-  const onWeeklyPickChange = async (
+  const handleWeeklyPickChange = async (
```

Comment:
Why was this name changed?

Reply:
I even changed it back to 'onWeeklyPickChange' and it just breaks things. I have to name the parent something different than the decoupled function I have inside

Reply:
Makes sense.